### ISSUE TITLE
[3.4.0-beta] OpenSCAD integration: Invalid file when loading OpenSCAD file

### ISSUE THREAD

**Issue by stelgenhof** [Ultimaker/Cura#3897]
**Application Version**
3.4.0-beta

**Platform**
macOS 10.13.5

**Printer**
Irrelevant

**Steps to Reproduce**
0. Install the OpenSCAD integration plugin (via Toolbox)
1. From the "File" menu, select "Open File(s)..."
2. Select any OpenSCAD file (with the extension .scad)

**Actual Results**
Error message "Invalid File: Failed to load <NAME OF FILE.SCAD>"

**Expected results**
With the OpenSCAD integration I was kind of hoping to be able to load an OpenSCAD file :) However I may be doing something incorrect, but couldn't find any instructions how this plugin need to be used.

Some documentation would be great :)

**Comment by Vandrasc** [comment]
Hi @stelgenhof, where is your openSCAD application located? Is it in the default installation folder? I experienced the same after I change the location of my application. Moving the openSCAD back to the default installation folder can do the trick. Let us know. 
Thank you!

**Comment by stelgenhof** [comment]
@Vandrasc My openSCAD is not in the default installation folder. Let me try moving the application back to its default installation folder and see if it then works.

**Comment by fieldOfView** [comment]
Pinging @thopiekar who wrote the plugin

**Comment by thopiekar** [comment]
Sadly I can't give any macOS support at all. That's simply because I don't own a Mac.
The only chance how to get the plugin working is to add the OpenSCAD binary's location to $PATH. Windows enjoys the little extra that the plugin will look up in the registry which application is assigned to the file. Apples default application framework provides such a functionality, too, but this would need another extra Python module in Cura built-in.

In short: I leak in hardware and software at this point.
With a lot of effort I could bring it up, but I'm too busy at the moment for this.

Excuse the bad news on this, but that how it currently is 🙁

Anyway, regards from Hilversum, Netherlands ✌️

**Comment by ChrisTerBeke** [comment]
Closing here as it's a plugin issue, not a Cura issue.

**Comment by Ghostkeeper** [comment]
See https://github.com/thopiekar/CuraOpenSCADPlugin for the source code (and issue reporting).
